### ISSUE TITLE
cannot connect to hpov global dashboard server via powershell

### ISSUE THREAD

**Issue by santhoshpcv** [HewlettPackard/POSH-HPOneView#563]
```powershell
Connect-HPOVMgmt : [Send-HPOVRequest]: The API version information is absent or invalid. Refer to the API
specification to determine the supported API versions.

+ ... onnection = Connect-HPOVMgmt -Hostname 172.25.77.50 -User $OVCredenti ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (Send-HPOVRequest:String) [Connect-HPOVMgmt], InvalidOperationExceptio
   n
    + FullyQualifiedErrorId : INVALID_API_VERSION,Connect-HPOVMgmt
```

I am getting this error please help.
I am using  hponeview.420 module version.

**Comment by VincentBerger** [comment]
You cannot connect to OneView **Global Dashboard** with the OneView Powershell module. Different products with different APIs. There is no equivalent Powershell module for Global Dashboard. If you absolutely need it, there is an old OV Global Dashboard module hosted [here](https://github.com/HewlettPackard/GlobalDashboardPS), but it is not endorsed by HPE in any way and hasn't been updated in almost 2 years.

**Comment by ChrisLynchHPE** [comment]
The HPEOneView Global Dashboard PowerShell library Vinent linked to is a library supported by HPE.  I am in the process of getting some needed OneView PowerShell library updates released since I have published the 6.00 library.  I will then begin work on updating the [GlobalDashboardPS ](https://github.com/HewlettPackard/GlobalDashboardPS)module to supporting the newer OVGD releases.

**Comment by ChrisLynchHPE** [comment]
Closing due to no further activity.  If you wish, feel free to re-open to discuss further.
